strobogrammatic-and-palindrome.html

```diff
@@ -1,74 +1,90 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <script src="https://cdn.tailwindcss.com"></script>
-    <title>Strobogrammatic and Palindrome</title>
-  </head>
-  <body>
-    <div class="container mx-auto">
-      <form class="mx-auto max-w-sm">
-        <div class="flex items-center border-b border-teal-500 py-2">
-          <input
-            class="appearance-none bg-transparent border-none w-full text-gray-700 mr-3 py-1 px-2 leading-tight focus:outline-none"
-            type="number"
-            placeholder="969"
-            aria-label="Enter a number"
-          >
-          <button
-            id="submit"
-            type="submit"
-            class="flex-shrink-0 bg-teal-500 hover:bg-teal-700 border-teal-500 hover:border-teal-700 text-sm border-4 text-white py-1 px-2 rounded"
-            type="button"
-          >
-            Check!
-          </button>
-        </div>
-      </form>
 
-      <div id="result" class="hidden">
-        <div class="mt-4">
-          <p class="text-gray-700 text-center">
-            Is it a Strobogrammatic number? <span id="strobogrammatic"></span>
-          </p>
-          <p class="text-gray-700 text-center mt-4">
-            Is it a Palindrome number? <span id="palindrome"></span>
-          </p>
-        </div>
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://cdn.tailwindcss.com"></script>
+  <title>Strobogrammatic and Palindrome</title>
+</head>
+
+<body
+  style=" height: 100vh; display: flex; justify-content: center; align-items: center; background-color: rgba(55, 59, 59, 0.411);">
```

Comment:
Please continue to use TailwindCSS for the rest of the project and updates
Ref:  https://tailwindcss.com/